packages/govuk-frontend-review/src/common/middleware/feature-flags.mjs

```diff
@@ -0,0 +1,57 @@
+import express from 'express'
+
+const router = express.Router()
+
+const COOKIE_NAME = 'use_recolour'
+
+/**
+ * Control cookie setting and unsetting when the setRecolour query is requested
+ *
+ * 3 possible states:
+ *
+ * 1. setRecolour == true: set the recolour cookie and redirect
```

Comment:
Does it make more sense if the param is `setUseRecolour`? Then you could get from the `COOKIE_NAME` const what the query param is if they were to follow the format `setCOOKIE_NAME`.

Reply:
Ah, unless you omit `use_` I suppose and `recolour` is the feature name. Never mind.